File: src/Models/Wines.java
package Models;

public class Wines implements HasID{
    public int wine_id;
    public String name;
    public int type_id;
    public int region_id;
    public int vintage_year;
    public int alcohol_percentage;
    public int supplier_id;

    public Wines(int wine_id, String name, int type_id, int region_id, int vintage_year, int alcohol_percentage, int supplier_id) {
        this.wine_id = wine_id;
        this.name = name;
        this.type_id = type_id;
        this.region_id = region_id;
        this.vintage_year = vintage_year;
        this.alcohol_percentage = alcohol_percentage;
        this.supplier_id = supplier_id;
    }

    public int getID() {
        return wine_id;
    }

    public void setWine_id(int wine_id) {
        this.wine_id = wine_id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getType_id() {
        return type_id;
    }

    public void setType_id(int type_id) {
        this.type_id = type_id;
    }

    public int getRegion_id() {
        return region_id;
    }

    public void setRegion_id(int region_id) {
        this.region_id = region_id;
    }

    public int getVintage_year() {
        return vintage_year;
    }

    public void setVintage_year(int vintage_year) {
        this.vintage_year = vintage_year;
    }

    public int getAlcohol_percentage() {
        return alcohol_percentage;
    }

    public void setAlcohol_percentage(int alcohol_percentage) {
        this.alcohol_percentage = alcohol_percentage;
    }

    public int getSupplier_id() {
        return supplier_id;
    }

    public void setSupplier_id(int supplier_id) {
        this.supplier_id = supplier_id;
    }

    @Override
    public String toString() {
        return "Wines{" +
                "wine_id=" + wine_id +
                ", name='" + name + '\'' +
                ", type_id=" + type_id +
                ", region_id=" + region_id +
                ", vintage_year='" + vintage_year + '\'' +
                ", alcohol_percentage='" + alcohol_percentage + '\'' +
                ", supplier_id=" + supplier_id +
                '}';
    }
}

File: Python code/Entities/Wines.py

class Wines:
    def __init__(self, wine_id, name, type_id, region_id, vinatage_year, alcohol_percentage, supplier_id):
        self.wine_id = wine_id
        self.name = name
        self.type_id = type_id
        self.region_id = region_id
        self.vinatage_year = vinatage_year
        self.alcohol_percentage = alcohol_percentage
        self.supplier_id = supplier_id

    def get_wine_id(self):
        return self.wine_id

    def get_name(self):
        return self.name

    def get_type_id(self):
        return self.type_id

    def get_region_id(self):
        return self.region_id

    def get_vinatage_year(self):
        return self.vinatage_year

    def get_alcohol_percentage(self):
        return self.alcohol_percentage

    def get_supplier_id(self):
        return self.supplier_id

    def set_wine_id(self, wine_id):
        self.wine_id = wine_id

    def set_name(self, name):
        self.name = name

    def set_type_id(self, type_id):
        self.type_id = type_id

    def set_region_id(self, region_id):
        self.region_id = region_id

    def set_vinatage_year(self, vinatage_year):
        self.vinatage_year = vinatage_year

    def set_alcohol_percentage(self, alcohol_percentage):
        self.alcohol_percentage = alcohol_percentage

    def set_supplier_id(self, supplier_id):
        self.supplier_id = supplier_id

    def __str__(self):
        return f"Wine: ID: {self.wine_id}, Name: {self.name}, Type ID: {self.type_id}, Region ID: {self.region_id}, Vintage Year: {self.vinatage_year}, Alcohol Percentage: {self.alcohol_percentage}, Supplier ID: {self.supplier_id}"